wagl/acquisition/__init__.py

```diff
@@ -35,6 +35,22 @@
 # resolution group format
 RESG_FMT = "RES-GROUP-{}"
 
+LANDSATMTLMAP = {
```

Comment:
Not a big fan of the condensed wording, but ok to move on with it.  As long as people are aware that there are 3 pieces to it:
* LANDSAT
* MTL
* MAP